### ISSUE TITLE
Voyant Tools Group 2: Mid-career Melville

### ISSUE THREAD

**Issue by kroncolleen** [SASDigitalHumanitiesTraining/Visualisation#13]
I removed all the stop words from the list except for the numbers and symbols (I thought these might be, say, page numbers.) 
I chose to include stop words in my analysis because of tutorial on Voyant tools, which suggested that authorship and style might be more closely related to the use of these small words than the sort of “theme” words. 
 
<img width="446" alt="Screen Shot 2020-06-17 at 10 29 39 PM" src="https://user-images.githubusercontent.com/55125873/84949733-f907d800-b0ed-11ea-9066-83adb731ff8f.png"> **(Stop words excluded)**

When I reintroduced the stop words, I noticed that “like” took on a much greater prominence. 

<img width="456" alt="Screen Shot 2020-06-17 at 10 12 45 PM" src="https://user-images.githubusercontent.com/55125873/84949808-16d53d00-b0ee-11ea-9463-3a521d28c8d7.png"> **(Stop words included)**

Although “like” had appeared in the stop-words excluded visualizations, it’s importance is clearer in the second/stop-word-included image. I guessed that this is due to Melville’s fondness for similes. I confirmed/denied this by looking at the “context” widget. Melville does, indeed, have some excellent similes: 
 
<img width="615" alt="Screen Shot 2020-06-17 at 10 26 13 PM" src="https://user-images.githubusercontent.com/55125873/84949921-4f751680-b0ee-11ea-9bf3-161e92b05b23.png">
<img width="594" alt="Screen Shot 2020-06-17 at 10 20 07 PM" src="https://user-images.githubusercontent.com/55125873/84949924-500dad00-b0ee-11ea-8159-e78a7b5439db.png">

In terms of “themes and ideas” I found this use of similes to be more interesting than, say, the thematic or dramatis personae type words (e.g., “Pierre,” “whale”). It’s not necessarily that interesting if “whale” shows up frequently in a book that purports to be about a whale. (Although perhaps it is worth pointing out that ‘Moby Dick’ as a phrase doesn’t even register.)
It’s more interesting (to me, anyway) to see the extent to which Melville relies on similes of the form “like a….” to provide thick description.

**Comment by Ellis307** [comment]
It might be a little bit late to reply here but I did have some thoughts in response to the discussion questions:

I did edit the stop word list by adding 'say', and ‘ye’ to stop words list. Initially, they still came through in the results. Once I tried again, however, they were then filtered out correctly. I thought it was useful to get rid of these as they are quite common words and I felt that they took away from the other themes being expressed in the Cirrus visualisation.

I thought it was also very interesting that the largest words in the Cirrus visualisation were Pierre (the titular character), whale, man, and various other thematically nautical words, e.g. boat, ship, captain. This was expected but it demonstrates that Melville was very clear in expressing his intended theme. There are also female names in there, who I assume are side characters because they are quite small in size. Another key thing I noticed was the use of introspective language, e.g. thought, soul, face, think, thought, know, mind.

**Comment by soneill-language-arts** [comment]
Voyant automatically excluded stop words when it first opened.
When I enabled stop words, it struck me that only male pronouns appeared in the top 55 most used words, but no female pronouns.
Excluding stop words, 'man' is the second most frequently used word in the corpus (1233 tokens)

I decided to explore the use of male and female pronouns in the corpus further + the use of some gendered terms (e.g. 'man' and 'woman')
Using the search feature, I explored the frequency  of male and female pronouns and reference terms further.
![Mid-Career Melville_Word Cloud_With Stop Words](https://user-images.githubusercontent.com/66736588/85008060-5898ce80-b154-11ea-9bd7-23bbf5dbe705.png)
![Term per work_male and female pronouns](https://user-images.githubusercontent.com/66736588/85008080-5f274600-b154-11ea-9239-fb11c5c182b0.png)

I visualised the male and female pronouns/ possessive adjectives using 'Trends' - I've tried exporting the graph, but the URL isn't working for me and unfortunately the Png (attached) does not display the key. From the values in the table you will be able to interpret the graph.


Term | Frequency | Term | Frequency
-- | -- | -- | --
He | 4062 | She | 638
Him | 2569 | Her | 1428
His | 5530 | Hers | 9

The 'Trends' widget shows that, by work, women and female pronouns feature most frequently in 'Pierre'. This text also contains prominent female characters, Isabel (395 tokens) and Lucy (300 tokens).
 
'Moby Dick' and 'The Confidence Man', by contrast, appear to inhabit male dominated spaces, in which women barely feature.
 
The 'Links' widget interestingly reveals that Isabel and Lucy most often appear linked to Pierre, i.e. the female characters most frequently appear in relation to the male protagonist.

![Links_Isabel Lucy and Pierre](https://user-images.githubusercontent.com/66736588/85008244-9dbd0080-b154-11ea-8203-0e37cbb31bb6.png)

A little bit of contextual work using the widget 'correlations' and the key word 'woman' showed that this term was used in conjunction with descriptions of appearance, crying and moral behaviour. I didn't have time to investigate this further or to examine the contextual list exhaustively, but this contextual use of the term 'woman' would appear to align with femininity tropes that could warrant further investigation.
